### DISCUSSION TITLE
Copilot never asked me to login (Invalid copilot token: missing token: 404)

### DISCUSSION THREAD

**Discussion by UltimateWalrus** [community/community#66149] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I can't use Github Copilot in Visual Studio Community 2022 17.7.3. I get the following:

GitHub Copilot 1.100.0.0 (v1.100.0.0@6ff082509)
GitHub Copilot Agent v1.100.0
[DEBUG] [agent] [2023-09-06T18:26:23.944Z] Agent service starting
[DEBUG] [agent] [2023-09-06T18:26:24.007Z] Telemetry initialized
Auth Status: NotSignedIn
[INFO] [auth] [2023-09-06T18:26:24.485Z] Invalid copilot token: missing token: 404 
[INFO] [auth] [2023-09-06T18:26:24.681Z] Invalid copilot token: missing token: 404 
Auth Status: NotAuthorized
You don't have access to GitHub Copilot.

It never even asked me to login to Github. I was expecting a dialogue box as explained in the "getting started" directions. Tried uninstalling and reinstalling the extension multiple times, and deleting the AppData/Local/github-copilot folder when I do.

**Comment by jemshit** [comment]
Check [this](https://stackoverflow.com/questions/76765315/visual-studio-invalid-copilot-token-missing-token-404)
